src/cli.js

```diff
@@ -208,6 +210,43 @@ export async function cli(args) {
 			break;
 		}
 
+		case 'badges': {
+			function showTemplateHelp() {
+				help({ command: 'help', args: ['badges'] });
+			}
+
+			if (!options.args.length) {
+				showTemplateHelp();
+				return;
+			}
+
+			const [command] = options.args;
+
+			switch (command) {
+				case 'init':
```

Comment:
I know this is how it has been working for awhile, but we should probably prevent attempts on specifying directories outside of the project.

<img width="681" alt="image" src="https://github.com/searchspring/snapfu/assets/5060400/2d052929-be87-480c-b330-9b26065efb43">

Reply:
<img width="687" alt="image" src="https://github.com/searchspring/snapfu/assets/5060400/d142193f-3a5f-4ee9-99f9-3818a9f95d02">